.changes/v2.15.0/436-improvements.md

```diff
@@ -0,0 +1 @@
+* Disk methods have now the ability to access new properties from API version 36.0. They are: `DiskRecordType.SharingType`, `DiskRecordType.UUID`, `DiskRecordType.Encrypted`, `Disk.SharingType`, `Disk.UUID` and `Disk.Encrypted`. Also, new method added `Disk.GetAttachedVmsHrefs [GH-436]
```

Comment:
> Also, new method added `Disk.GetAttachedVmsHrefs [GH-436]

This new feature is unrelated to the new fields? If yes, it deserves a separate bullet point in the changelog.

Reply:
changed